synapse/rest/internal/client/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not entirely convinced that `internal` is the right name here, though I must admit I'm struggling to come up with better alternatives.

Reply:
I've renamed it to `synapse` (which requires a modification to an import in `synapse/rest/__init.py`. Does this a) sound good b) seem sane?